cloudfoundry-client-spring/src/main/lombok/org/cloudfoundry/spring/client/v2/organizationquotadefinitions/SpringOrganizationQuotaDefinitions.java

```diff
@@ -56,6 +57,11 @@ public SpringOrganizationQuotaDefinitions(RestOperations restOperations, URI roo
     }
 
     @Override
+    public Mono<Void> delete(DeleteOrganizationQuotaDefinitionRequest request) {
+        return delete(request, Void.class, builder -> builder.pathSegment("v2", "quota_definitions", request.getOrganizationQuotaDefinitionId()));
```

Comment:
@antechrestos These aren't `Void`.  Since they have an `async` argument, it needs to match [this style](https://github.com/cloudfoundry/cf-java-client/blob/master/cloudfoundry-client-spring/src/main/lombok/org/cloudfoundry/spring/client/v2/organizations/SpringOrganizations.java#L162-L168)